doc/changes/latest.inc

```diff
@@ -71,7 +71,11 @@ Enhancements
 
 - Show multiple colors and linestyles for excluded components with :class:`mne.Evoked` in :meth:`mne.preprocessing.ICA.plot_sources` (:gh:`9444` by `Martin Schulz`_)
 
-- Added tutorial for how to processes image (CT and MR) files in order to localize electrode contacts for intracranial recordings :ref:`tut-ieeg-localize` (:gh`9484` by `Alex Rockhill`_)
+- Add functions for aligning MRI and CT data `mne.transforms.compute_volume_registration` and `mne.transforms.apply_volume_registration` (:gh:`9503` by `Eric Larson`_ and `Alex Rockhill`_)
```

Comment:
```suggestion
- Add functions for aligning MRI and CT data :func:`mne.transforms.compute_volume_registration` and :func:`mne.transforms.apply_volume_registration` (:gh:`9503` by `Eric Larson`_ and `Alex Rockhill`_)
```

Reply:
We don't need this anymore (our default role is `:py:obj`) and at least some people have expressed a preference for not doing it